src/openapi.yaml

```diff
@@ -221,13 +220,58 @@ paths:
               $ref: '#/components/schemas/Zaak'
         required: true
     parameters: []
+  /zaken/_zoek:
+    post:
+      operationId: zaak__zoek
+      description: Voer een (geo)-zoekopdracht uit op ZAAKen.
+      parameters:
+        - name: Accept-Crs
+          in: header
+          description: >-
+            Het 'Coordinate Reference System' (CRS) van de verzoekdata. Volgens
+            de GeoJSON spec is WGS84 de default.
+          required: true
+          schema:
+            type: string
+            enum:
+              - 'EPSG:4326'
+      responses:
+        '200':
+          description: ''
+          headers:
+            Content-Crs:
+              description: >-
+                Het 'Coordinate Reference System' (CRS) van de antwoorddata.
+                Volgens de GeoJSON spec is WGS84 de default.
```

Comment:
Wellicht goed om aan te geven dat WGS84 hetzelfde is als EPSG:4326, door dit bijv. tussen haakjes er achter te zetten.

https://en.wikipedia.org/wiki/World_Geodetic_System